admin/src/main/java/io/strimzi/arguments/topic/AlterTopicCommand.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.arguments.topic;
+
+import io.strimzi.admin.AdminProperties;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.NewPartitions;
+import picocli.CommandLine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@CommandLine.Command(name = "alter")
+public class AlterTopicCommand extends BasicTopicCommand {
```

Comment:
Add some JavaDoc what this class provides and what the purpose is if one decides to extend it somehow.